tests/SprykerEcoTest/Zed/Payone/Business/Facade/PayoneFacadeDistributePricesTest.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEcoTest\Zed\Payone\Business\Facade;
+
+use Codeception\Test\Unit;
+use Generated\Shared\Transfer\ItemTransfer;
+use Generated\Shared\Transfer\OrderTransfer;
+use SprykerEcoTest\Zed\Payone\Business\DataProvider\PayoneDistributePricesDataProvider;
+
+/**
+ * @group Unit
+ * @group SprykerEco
+ * @group Zed
+ * @group Payone
+ * @group Business
+ * @group Facade
+ * @group PayoneFacadeDistributePricesTest
+ */
+class PayoneFacadeDistributePricesTest extends Unit
+{
+    use PayoneDistributePricesDataProvider;
+
+    protected const PAYMENT_PROVIDER_PAYONE = 'Payone';
+    protected const PAYMENT_PROVIDER_DUMMY = 'Dummy';
+
+    /**
+     * @var \SprykerEcoTest\Zed\Payone\PayoneZedTester
+     */
+    protected $tester;
+
+    /**
+     * @dataProvider provideOrdersData
+     *
+     * @param array $orderItemsData
+     * @param array $expensesData
+     * @param array $paymentsData
+     * @param array $totalData
+     * @param int $expectedTotalPrice
+     *
+     * @return void
+     */
+    public function testDistributePricesShouldReturnOrderTransferWithDistributedPrices(
```

Comment:
Please add the test cases to the existing affected facade methods to check the price distribution.

Reply:
Discussed and fixed